twenty-first/src/shared_math/polynomial.rs

```diff
@@ -2294,7 +2304,7 @@ mod test_polynomials {
     }
 
     #[test]
-    #[should_panic(expected = "assertion `left != right` failed")]
+    #[should_panic(expected = "Line must not be parallel to y-axis")]
```

Comment:
Interesting! For some reason, all tests had passed for me. CI also did not mention any broken tests.

Reply:
Weird! I learned, though, that `expected` only checks for substrings in the panic message. It doesn't do equality check on the two strings.